user-lock-manager.smartapp.groovy

```diff
@@ -1066,3 +1119,69 @@ private sendMessage(msg) {
     sendNotificationEvent(msg)
   }
 }
+
+def checkRecentUser() {
+  debugLog("checkRecentUse: Checking state.cookieCurrentUser: ${state}")
+  if ( state.cookieCurrentUser ) {
+    debugLog "Returned from user ${state.cookieCurrentUser} settings page"
+    def id = state.cookieCurrentUser
+    if ( ! id.isNumber() ) {
+    	debugLog("${id} is NOT a number")
+        id = state.cookieCurrentUser.toInteger()
+    } 
+    if ( id > 0 ) {
+	debugLog("Processing slot #$id")
+	processUserCode(id)
+    }
+  }
+}
+
+def processUserCode(id) {
```

Comment:
This function doesn't take into account schedules, and will activate a code that otherwise should not be active.  I would allow the regular initialization and polling interval to take care of this for us instead of initializing users individually.